spec/highlighting.spec.js

```diff
@@ -54,4 +76,237 @@ describe('Highlighting', function () {
     })
   })
 
+  describe('highlightSupport', () => {
+    beforeEach(() => {
+      setupHighlightEnv(this, 'People Make The <br> World Go Round')
```

Comment:
We should replace all arrow functions in this file with regular functions if you use `this` here. Otherwise this code is quite misleading.

Reply:
(We could also do this in a separate PR for all tests in editable.js)

Reply:
I would like to go with your second suggestion, as currently a *lot* of tests misbehave if you only just adjust the `beforeEach` scopings.

Reply:
> I would like to go with your second suggestion

That's ok

> as currently a lot of tests misbehave if you only just adjust the beforeEach scopings.

This is because you have to change all methods in the same file at once, otherwise you mix scopes which does not work.

Reply:
I did replace all of them to the point where I had now arrow functions at all anymore (at least in this spec), but still all hell broke loose. I'd rather do this with a little more focus and time.